packages/orbit-components/src/tmp-tailwind/InputField/index.tsx

```diff
@@ -0,0 +1,318 @@
+"use client";
+
+import * as React from "react";
+import cx from "clsx";
+
+import { TYPE_OPTIONS } from "../../InputField/consts";
+import InputTags from "./InputTags";
+import getFieldDataState from "../../common/getFieldDataState";
+import useRandomId from "../../hooks/useRandomId";
+import ErrorFormTooltip from "../../ErrorFormTooltip";
+import AlertCircle from "../../icons/AlertCircle";
+import InformationCircle from "../../icons/InformationCircle";
+import FormLabel from "../../FormLabel";
+import useErrorTooltip from "../../ErrorFormTooltip/hooks/useErrorTooltip";
+import type { Props } from "../../InputField/types";
+import { spaceAfterClasses } from "../../common/tailwind";
+
+const getDOMType = (type: Lowercase<keyof typeof TYPE_OPTIONS>) => {
+  if (type === TYPE_OPTIONS.PASSPORTID) return TYPE_OPTIONS.TEXT;
+  return type;
+};
+
+export const fakeInputSnippet = ({
```

Comment:
Could we have just a simple `FakeInput` component (as we used to have) rendering a div with these classes? Or is there any reason why this is has to be function?

Reply:
I think with Tailwind it's better for such cases to use the function as if needed, you can adjust it, use any element you want etc., more flexible.

Reply:
I agree it's more flexible, but this doesn't necessarily mean that's better. Following that criterion, we should make all simple components the same way, and I don't recall seeing this kind of function around, but I might be wrong.

In addition, we could potentially introduce an `as` prop defaulting to `div` if we need this flexibility.

Reply:
I do not think we should continue arguing about that, it helps you to achieve what you want and it can be in form of a component or a function. Feel free to refactor it to a component, if you want later in your request. I do not think adding 'as' prop will be a good thing, this looks simpler to me. Let's not block this MR more :)

Reply:
I'm not blocking it, just expressing my opinion.

> it helps you to achieve what you want

This is not about what I want, it's not personal. I'm just discussing what I believe is better for the project. But you're right, it's not worth arguing more.

In any case, this PR will break the Select component once is moved to the src folder, I guess this is a downsize of using a tmp folder instead of branches. But no worries, I'll adapt it on my side.

Reply:
I can't agree with this being a function. This was, and should continue to be, a component. It receives props and renders an HTML element with classes and some logic based on those props. That's the React abstraction working.
The `Select` component uses this and should not need to worry about its internal implementation, including which HTML element it uses. The only thing users of this FakeInput should know is its class so they can target it: `.orbit-input-field-fake-input` (as it is right now, but it can be any other thing, like simply `.orbit-fake-input`).

Furthermore, this FakeInput is using `peer` related classes. If it's a reusable component (or function), at first, I am not sure if it is a good approach (I know I was the one suggestind it at first, but the context is now broader... and it eventually can play in our favor). This means that if we use this as a sibling of any other component with `peer` we might get undesirable side effects. However, and looking at the current scenario, it seems it can actually play in our favor as we have things right now:
The `Select` component can take advantage of this and simplify its logic by removing all the targets to the `.orbit-input-field-fake-input` class. Applying the `peer` class to the `select` element rendered inside should be enough to ensure the very same visual styles that we have in InputField, which is what we want.

So, to me, this should be a component. And for now it is kept inside this `InputField` folder with the proper class and the peer targets, as we have. And we will be able to migrate Select taking advantage of the class selector and the peer classes that already exist.
In the future, we can (probably should) reconsider extracting this component to its own folder, with its own tests, and independent implementation. And we can evaluate the effort of not relying on `peer` classes on a component that is meant to be reused in more than one place. This does not mean we will export the component, but it can be kept as an internal component that is still capable of livingo on its own and suits the needs of whoever uses it.

Reply:
I will refactor it to the component in order just to stop this discussion and continue with migration, but all these arguments above don't convince me that it should be a component

_> It receives props and renders an HTML element with classes and some logic based on those props._
There's not much of a difference between receiving a prop or function parameter in this situation. 

Before it was the component just because you can't do it the other way with SC. Now we can't trigger that via class names and such things can be considered as snippets or mixins that can be applied with function and extended or slightly changed if needed (with the component you can't do that) 

This whole paragraph about the peer is interesting, but not related to this situation, isn't it? 
 
 > In the future, we can (probably should) reconsider extracting this component to its own folder, with its own tests, and independent implementation.  
 
No, we shouldn't. Tested should be only input/select native elements, those visuals do not need to be abstracted as components and tested. They will be tested in visual regression within the whole component.